flink-table/flink-table-planner/src/test/java/org/apache/flink/table/planner/plan/stream/sql/MultiJoinTest.java

```diff
@@ -684,4 +695,705 @@ void testMultiSinkOnMultiJoinedView() {
+     * Therefore, in this test, each Join is still converted to a MultiJoin individually.
+     */
+    @Test
+    @Tag("multijoin-chain-expected")
+    void testJoinConditionHasNestedFieldsMultiJoinChainExpected() {
+        util.tableEnv()
+                .executeSql(
+                        "CREATE TABLE Developers ("
+                                + "  developer_id STRING PRIMARY KEY NOT ENFORCED,"
+                                + "  person ROW<info ROW<id STRING, name STRING, region STRING>>,"
+                                + "  experience_years INT"
+                                + ") WITH ('connector' = 'values', 'changelog-mode' = 'I,UA,D')");
+
+        util.tableEnv()
+                .executeSql(
+                        "CREATE TABLE SupportTickets ("
+                                + "  ticket_id STRING PRIMARY KEY NOT ENFORCED,"
+                                + "  reporter ROW<info ROW<id STRING, priority STRING>>,"
+                                + "  issue STRING"
+                                + ") WITH ('connector' = 'values', 'changelog-mode' = 'I,UA,D')");
+
+        util.tableEnv()
+                .executeSql(
+                        "CREATE TABLE Feedback ("
+                                + "  feedback_id STRING PRIMARY KEY NOT ENFORCED,"
+                                + "  author ROW<info ROW<id STRING, rating INT>>,"
+                                + "  message STRING"
+                                + ") WITH ('connector' = 'values', 'changelog-mode' = 'I,UA,D')");
+
+        util.tableEnv()
+                .executeSql(
+                        "CREATE TABLE Subscriptions ("
+                                + "  sub_id STRING PRIMARY KEY NOT ENFORCED,"
+                                + "  subscriber ROW<info ROW<id STRING, plan STRING>>,"
+                                + "  active BOOLEAN"
+                                + ") WITH ('connector' = 'values', 'changelog-mode' = 'I,UA,D')");
+
+        util.verifyRelPlan(
+                "SELECT "
+                        + "  d.developer_id, "
+                        + "  d.person.info.name AS developer_name, "
+                        + "  s.ticket_id, "
+                        + "  s.reporter.info.priority AS ticket_priority, "
+                        + "  f.feedback_id, "
+                        + "  f.author.info.rating AS feedback_rating, "
+                        + "  sub.sub_id, "
+                        + "  sub.subscriber.info.plan AS subscription_plan "
+                        + "FROM Developers AS d "
+                        + "LEFT JOIN SupportTickets AS s "
+                        + "  ON d.person.info.id = s.reporter.info.id "
+                        + "LEFT JOIN Feedback AS f "
+                        + "  ON d.person.info.id = f.author.info.id "
+                        + "LEFT JOIN Subscriptions AS sub "
+                        + "  ON d.person.info.id = sub.subscriber.info.id");
+    }
```

Comment:
Can you please here and in other sql insert EOLs
otherwise it is required to scroll a lot just to see the query in plan xml